platform/darwin/src/MGLLight.h

```diff
@@ -0,0 +1,109 @@
+#import <CoreGraphics/CoreGraphics.h>
+
+#import "MGLFoundation.h"
+#import "MGLStyleValue.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+
+/** Options to specify extruded geometries are lit relative to the map or viewport. */
+typedef NS_ENUM(NSUInteger, MGLLightAnchor) {
+    /** The position of the light source is aligned to the rotation of the map. */
+    MGLLightAnchorTypeMap,
```

Comment:
I think these should match the base name so you should remove `Type`. So `MGLLightAnchor` is in both `MGLLightAnchorMap` and `MGLLightAnchorViewport`